### ISSUE TITLE
Feature score hint seems wrong

### ISSUE THREAD

**Issue by SebastianZ** [Devographics/surveys#248]
I just completed the State of CSS 2024 survey, and I was presented with this:

![Features Score](https://github.com/user-attachments/assets/cada5d50-781c-4c82-a051-dfa6b6843bf4)

It says that I am in the "top 100%". I expected I'd be. 😅

I assume that number is incorrectly calculated, as it indicates that all participants have a score that high or higher. And I truly believe there are at least a few participants so far that _don't know_ about at least one feature.

Though even when the number is correct (e.g. due to being the first participant or all other participants scoring higher), the hint shouldn't be displayed if the number is 100%. Maybe it should only be displayed if you're above average.
Or there could even be different hints for different scores, encouraging low-scoring participants to read more about the features and congratulating high-scoring ones.

Sebastian

**Comment by SachaG** [comment]
Yeah, I think we'll end up scrapping the whole system because calculating that percentage requires running costly (and evidently, wrong) db queries every time. 

Or maybe we should just do a `rand(100)` :p
